marketplace/deployer_util/config_helper.py

```diff
@@ -33,6 +33,7 @@
 XTYPE_SERVICE_ACCOUNT = 'SERVICE_ACCOUNT'
 XTYPE_STORAGE_CLASS = 'STORAGE_CLASS'
 XTYPE_STRING = 'STRING'
+XTYPE_APPLICATION_UID = 'APPLICATION_UID'
```

Comment:
Add validation that this field must be specified for helm-tiller deployer. Please follow up in another PR.